templates/fine-tune-GPTJ/cluster_env.yaml

```diff
@@ -0,0 +1,20 @@
+# See https://hub.docker.com/r/anyscale/ray for full list of
+# available Ray, Python, and CUDA versions.
+base_image: anyscale/ray-ml:nightly-py310-gpu
```

Comment:
Is it possible to just build an image off of `ray` instead of `ray-ml`? I'd rather just use the lightweight one when possible (smaller image size, fewer dependency conflicts with other random packages that users won't need for this template).